doc/model/axes/axis-cartesian-continuous.xml

```diff
@@ -67,6 +67,23 @@
             </c:documentation>
         </c:property>
 
+        <c:property name="ticksRegularity"
+                    type="pvc.options.varia.AxisTicksRegularity"
```

Comment:
You sould use the `boolean` type directly. We should not specify a default if it comes to be dynamic, as I suggest above, depending on the value of the sliding window option.